stingray/largememory.py

```diff
@@ -558,9 +559,8 @@ def _addSpectra(final_spectra, curr_spec, flag):
 
         return final_spectra
 
-    np.multiply(np.add(final_spectra.freq, curr_spec.freq),
-                curr_spec.m,
-                out=final_spectra.freq)
+    assert np.allclose(final_spectra.freq, curr_spec.freq), \
```

Comment:
Isn't there a need to add and multiply the frequency?

Reply:
No. The frequency needs to be identical in all chunks

Reply:
But what about the frequency of the final_added combined spectra?

Reply:
Nope.
You are averaging, not concatenating, spectra. You average each bin corresponding to each frequency in each spectrum.  The final spectrum has exactly the same frequencies of all of the sub-spectra

Reply:
Ohhh, yeah, okay then!!